zdm/iteration.py

```diff
@@ -852,21 +720,24 @@ def missing_cube_likelihoods(grids,surveys,todo,outfile,norm=True,psnr=True,star
     f.close()
 
 
-def cube_likelihoods(grids,surveys, #psetmins,psetmaxes,npoints,
+def cube_likelihoods(grids:list,surveys:list, 
```

Comment:
Hey @profx as a demonstrator to this, it'd be good to have a quick run where the cube routine does not update the grid (i.e. it runs very quickly!) but the cubelikelihoods *does* iterate and spit out the parameters of each iteration. Just to ensure it is behaving properly. I assume you've done a check of this behaviour in suitably many dimensions?

Reply:
Let's discuss.  Not sure I follow.